### ISSUE TITLE
Button with borderRadius prop crash on API < 21

### ISSUE THREAD

**Issue by jjdp** [react-native-training/react-native-elements#597]
Buttons with a border radius like so only works on Lollipop and above because of the ripple drawable is not available.

`<Button
              title="Login"
              borderRadius={5}
              onPress={() => this._login()}/>`

https://github.com/react-native-training/react-native-elements/blob/32bf779930f928bf2a22658dcb9ca729535610d1/src/buttons/Button.js#L106

We should add an if API > 21 on that and add a proper borderRadius @Monte9

**Comment by Monte9** [comment]
@jjdp good catch. Agreed, we should make it conditional so that it doesn't crash for API < 21

Would you be willing to submit a PR for it? I'd gladly review and merge it in.

**Comment by jjdp** [comment]
#607

**Comment by iRoachie** [comment]
Just merged in https://github.com/react-native-training/react-native-elements/pull/616/commits/d59711d6c25078a3f3088bfd81e03eb55221cdf1. Will be available next release

**Comment by timscott** [comment]
This works as a hack:

```jsx
<Button
  ...
  background={TouchableNativeFeedback.SelectableBackground()}
/>
```

**Comment by iRoachie** [comment]
Fixed in v18.0.0
